spec/unit/plugin_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'shared_contexts'
+require '3scale_toolbox/cli'
+
+RSpec::Matchers.define_negated_matcher :not_raise_error, :raise_error
+
+RSpec.describe 'Plugin command' do
+  include_context :temp_dir
+  include_context :plugin
+  include_context :random_name
+
+  around(:each) do |example|
+    $LOAD_PATH.unshift(tmp_dir) unless $LOAD_PATH.include?(tmp_dir)
+    example.run
+    $LOAD_PATH.delete(tmp_dir)
+  end
+
+  it 'is not loaded when not in load path' do
+    expect do
+      ThreeScaleToolbox::CLI.run(%w[simple])
+    end.to raise_error.and output(/unknown command/).to_stderr
```

Comment:
Is it expected to exit? If so, then I guess you should have a matcher that says something `exit_with(1)` to be explicit it exits with some status code.